Add Ports explanation to troubleshootingp page (#309)
CleverCloud/documentation: content/doc/find-help/troubleshooting.md

Check the status of the services on [clevercloudstatus.com](https://www.clevercloudstatus.com/). The support team is also there to help you identify any potential errors or misconfiguration.
{{% /details %}}

> {{% details title="A lot of port appear open after a scan of my domain" closed="true" %}}

Reviewer: ```suggestion
{{% details title="There are a lot of open ports, according to a port scan." closed="true" %}}
```

If you tried to scan your domain, you may not only have scanned your app, but also several public load balancers.
Here is an explanation of the different port you may see.

> - Port 9999 : Zabbix use this port to monitor the hypervisor.

Reviewer 2: Fix grammar and add link to Zabbix homepage, so readers know what we are talking about.

```suggestion
- Port 9999 : [Zabbix](https://www.zabbix.com) uses this port to monitor the hypervisor.
```

> - Ports 5000 to 6000 : The TCP redirection feature use these port to work. If you see the port in this range open but don't have the feature enabled, it means other app also present on the same hypervisor uses this feature. The load balancer link a specific port to an app, you don't have to worry about other people using this port to access your app.

Reviewer 2: ```suggestion
- Ports 5000 to 6000 : The TCP redirection feature uses these ports to work. If you notice a port within this range open but have not enabled the feature, it indicates that another app on the same hypervisor is using it. The load balancer assigns a unique port to each app, ensuring that your app is securely isolated. You can be assured that other users cannot access your app through this port.
```

- Port 3000 : Juggernaut, the daemon torrent for the VM images use this port.

> - Ports 1080 and 1081 : They are necessary for the reverse-proxy Sozu to be able to redirect traffic to your application. Port 1080 is for HTTP and 1081 for HTTPS

Reviewer: ```suggestion
- Ports 1080 and 1081 : Those are necessary for the reverse-proxy Sōzu to be able to redirect traffic to your application. Port 1080 is for HTTP and 1081 for HTTPS
```

> - Port 80 and 443 : Your app use these ports to receive HTTP and HTTPS traffic

Reviewer 2: ```suggestion
- Port 80 and 443 : Your app uses these ports to receive HTTP and HTTPS traffic
```
